gems/jquery-rails/CVE-2015-9251.yml

```diff
@@ -0,0 +1,23 @@
+---
+gem: jquery-rails
+framework: rails
+cve: 2015-9251
+ghsa: rmxg-73gg-4p98
+url: https://github.com/rails/jquery-rails/blob/master/CHANGELOG.md#420
```

Comment:
This github URL should be locked to a version tag. ChangeLogs always grow downwards with new releases added to the top.

Reply:
Changed to https://github.com/rails/jquery-rails/releases/tag/v4.2.0